 *******> update.8

 Author: Stephan Schott-Verdugo

 Date: Fabruary 10, 2025

 Programs: packmol-memgen

 Description: Force using propka for titration with_ph in pdb2pqr

 --------------------------------------------------------------------------------

 AmberTools/src/packmol_memgen/packmol_memgen/lib/utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git AmberTools/src/packmol_memgen/packmol_memgen/lib/utils.py AmberTools/src/packmol_memgen/packmol_memgen/lib/utils.py
index 0b3763036c..5cd4a2817d 100755
--- AmberTools/src/packmol_memgen/packmol_memgen/lib/utils.py
+++ AmberTools/src/packmol_memgen/packmol_memgen/lib/utils.py
@@ -53,7 +53,7 @@ def pdb2pqr_protonate(pdb,overwrite=False,ffout='AMBER',pH=7.0):
 #        pdb2pqr_parser = pdb2pqr_args()
 #        args = pdb2pqr_parser.parse_args(['--pdb-output='+output_pdb,'--ff=PARSE','--ffout=AMBER','--with-ph='+str(pH),pdb,output_pqr])
 #        pdb2pqr(args)
-    args = " ".join([pdb2pqr,'--pdb-output='+output_pdb,'--ff=PARSE','--ffout=AMBER','--with-ph='+str(pH),pdb,output_pqr,"&> pdb2pqr.log"])
+    args = " ".join([pdb2pqr,'--pdb-output='+output_pdb,'--ff=PARSE','--ffout=AMBER','--titration-state-method=propka','--with-ph='+str(pH),pdb,output_pqr,"&> pdb2pqr.log"])
     rtrn_code = os.system(args)
     if rtrn_code != 0:
         logger.critical("CRITICAL:\n PDB2PQR failed! Check pdb2pqr.log")
